### PR TITLE
doc: update the meeting schedule in readme

### PR DESCRIPTION
**Pull request by riya-singhal31**
Updated the meeting schedule in readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,8 +180,8 @@ More details are available [here](https://github.com/ceph/ceph-csi/issues/463)
 
 ## Dev standup
 
-A regular dev standup takes place every [Monday,Tuesday and Thursday at
-12:00 PM UTC](https://meet.google.com/nnn-txfp-cge). Convert to your local
+A regular upstream dev standup takes place every [Tuesday at
```
Comment (nixpanic) [inline README.md]: this is a community project, there is no need to make a difference between upstream/downstream as it is per definition about upstream :smiley:

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (riya-singhal31) [reply README.md]: Thanks for the review Niels, updated.

**Review by nixpanic — APPROVED** [review]

**Review by Rakshith-R — APPROVED** [review]

**Review by humblec — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio requeue

**Comment by nixpanic** [comment]
@riya-singhal31 could you please login on the [Mergify Dashboard](https://dashboard.mergify.com) so that Mergify can rebase and merge this?

See https://github.com/ceph/ceph-csi/pull/3695/checks?check_run_id=11911820072 for the message.

**Comment by nixpanic** [comment]
@mergifyio refresh

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by riya-singhal31** [comment]
@Mergifyio refresh

**Comment by riya-singhal31** [comment]
@Mergifyio rebase

**Comment by nixpanic** [comment]
@mergifyio rebase

**Comment by nixpanic** [comment]
@requeue

**Comment by nixpanic** [comment]
@mergifyio requeue

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
@mergifyio rebase

**Pull request merged**